### ISSUE TITLE
Release dance for level@5

### ISSUE THREAD

**Issue by vweevers** [Level/community#56]
Prerelease, in topological order:

* [x] **[`level@next`](https://github.com/Level/level)**  (using new `level-packager` and `leveldown`)
  * [x] **[`level-packager@5`](https://github.com/Level/packager)** (using new  `levelup` and `encoding-down`) 
    * [x] **[`encoding-down@6`](https://github.com/Level/encoding-down)** (due to `abstract-leveldown@6`) 
      * [x] Remove `_setupIteratorOptions` (https://github.com/Level/encoding-down/pull/74)
      * [x] Prepare changelog and upgrade guide (https://github.com/Level/encoding-down/pull/73)
      * [x] Release (https://github.com/Level/encoding-down/issues/71)
    * [x] **[`levelup@4`](https://github.com/Level/levelup)** (using new  `level-iterator-stream` and `deferred-leveldown`)
      * [x] **[`level-iterator-stream@4`](https://github.com/Level/iterator-stream)** (due to `readable-stream@3`)
      * [x] **[`deferred-leveldown@5`](https://github.com/Level/deferred-leveldown)** (due to `abstract-leveldown@6`)
      * [x] Remove IE10 from Airtap (due to `readable-stream@3`) (https://github.com/Level/levelup/pull/625)
      * [x] Release (https://github.com/Level/levelup/issues/623)
    * [x] Release (https://github.com/Level/packager/issues/78)
  * [x] **[`leveldown@next`](https://github.com/Level/leveldown)** (due to `abstract-leveldown@6` + N-API rewrite)
    * [x] Release RC `5.0.0-0`
    * [x] Patch `prebuildify` to handle custom targets while using `--napi` (https://github.com/mafintosh/prebuildify/pull/17)
    * [x] Patch `node-gyp-build` to handle `--build-from-source` (https://github.com/Level/leveldown/issues/566)
    * [x] Release RC `5.0.0-1`
  * [x] Release (https://github.com/Level/level/issues/132)

Stable:

* [x] **[`level@5`](https://github.com/Level/level)**  (using new `leveldown`)
  * [x] ~Add more tests to  **[`encoding-down`](https://github.com/Level/encoding-down)**~
  * [x] **[`leveldown@5`](https://github.com/Level/leveldown)**
    * [x] Release (https://github.com/Level/leveldown/issues/550)
  * [x] Release (https://github.com/Level/level/issues/123)

Work that can happen in parallel but is not necessarily blocking:

- [x] Reorder v6 upgrade guide (https://github.com/Level/abstract-leveldown/issues/317)
- [ ] Reach out to and/or canary test dependents
   - [ ] ssb and flume: https://github.com/ssbc/ssb-server/issues/598#issuecomment-449559560
   - [x] `fastify-leveldb`: https://github.com/fastify/fastify-leveldb/pull/18#issuecomment-449534322
   - [ ] PouchDB?
   - [ ] others?
- [x] Update [`awesome`](https://github.com/Level/awesome) (https://github.com/Level/awesome/commit/1602c2c66df668def9bd2d59e91d7c71e4a9e3ee)

**Comment by ralphtheninja** [comment]
One approach to this could be:

* release `encoding-down` as is (without the extra tests)
* release a new `next` version of `leveldown`
* release a new `next` version of `level`
* try out the `next` version of `level` in important dependents to get feedback
* add tests to `encoding-down` based on feedback
* release `leveldown`
* release `level`

**Comment by vweevers** [comment]
@ralphtheninja Sounds good, I think. I need a little bit of time to wrap my head around it :)

**Comment by vweevers** [comment]
OK let's go for it. Having a `next` version of `level` will make testing much easier.

I've updated the tasks above.

**Comment by vweevers** [comment]
Phew. Done!
